TSOClient/tso.client/UI/Panels/UIChatDialog.cs

```diff
@@ -10,6 +10,12 @@
 
 namespace FSO.Client.UI.Panels
 {
+    public static class SizeFactor
+    {
+        public const int Decrease = -1;
+        public const int Increase = 1;
+    }
```

Comment:
No need to make this an enum - for simple deltas like 1 and -1, it makes more sense to just pass in the delta unnamed.

For future reference, though, enums in C# work as follows
```csharp
    public static enum SizeFactor
     {
         Decrease = -1,
         Increase = 1
     }
```

Reply:
Some people like it for documentation purpose, I figured I was being clever just in-case you wanted it to be self-documenting. I decided to not use the enum because I didn't want to cast (int)factor later on, I'll reduce this to just the delta.